src/Components/HomePage/HomePage.js

```diff
@@ -6,9 +6,11 @@ const HomePage = () => {
     <section className='main-container columns'>
       <aside className='mission'>
         <h2>Our Mission</h2>
-        <p>To build connections between parents of baby’s in need and generous donors.</p>
-        <p>Additional messaging about our mission... </p>
-        <p>Message about providing resources and support..... Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation.</p>
+        <p>Our orgnanization strives to build connections between parents of babies in need and generous donors.</p>
+        <p>Human milk is the natural food for babies and provides them with optimal nutrition that allows them to reach their health potential.
+           We are here to help mothers who are not able to nurse their babies or produce enough milk by connecting them with mothers who produce more milk than their babies need. 
+           Together, we will help infants thrive by making donated breastmilk safe and accessible.</p>
+        <p>We understand that breastfeeding issues can lead to many different stressers and emotions. We are here for you and you are not alone. Please visit our Support page to view more resources.</p>
```

Comment:
stressers -> stressors

Replace first sentence with the following:
The period of time after a baby is born is naturally a very stressful time. Emotions and pressure is high. We understand that breastfeeding issues can be a major contributor to a mother's stress level.